tests/helpers/start-app.js

```diff
@@ -5,8 +5,8 @@ import config from '../../config/environment';
 export default function startApp(attrs) {
   let application;
 
-  let attributes = Ember.merge({}, config.APP);
-  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
+  let attributes = Ember.assign({}, config.APP);
```

Comment:
@rwjblue we are not using destructuring yet in any other blueprint, IMHO that should be done either for all or for none to stay consistent. also the guides should match whatever style we're suggesting.

/cc @locks